hasher-matcher-actioner/webapp/src/pages/ContentDetailsWithStepper.tsx

```diff
@@ -87,10 +87,10 @@ export default function ContentDetailsWithStepper(): JSX.Element {
     <FixedWidthCenterAlignedLayout title="Progress">
       <Row>
         <Col md={{span: 6}}>
-          {contentPreviewURL ? (
+          {contentPreviewURL && contentType ? (
             <ContentPreview
               contentId={id}
-              contentType={ContentType.Photo}
+              contentType={getContentTypeForString(contentType!)}
```

Comment:
aside: curious, what does the `!` do?

Reply:
type casts optional<T> to T. Also, will throw a runtime error if contentType is undefined. Hence the check in line 90.